src/main/java/io/github/thebusybiscuit/sensibletoolbox/items/EnderLeash.java

```diff
@@ -247,36 +274,43 @@ private void thawEntity(Animals entity, Configuration conf) {
             ((Pig) entity).setSaddle(conf.getBoolean("saddled"));
             break;
         case SHEEP:
-            ((Sheep) entity).setSheared(conf.getBoolean("sheared"));
-            ((Sheep) entity).setColor(DyeColor.valueOf(conf.getString("color")));
+            Sheep sheep = (Sheep) entity;
+            sheep.setSheared(conf.getBoolean("sheared"));
+            sheep.setColor(DyeColor.valueOf(conf.getString("color")));
             break;
-        case OCELOT:
-            ((Ocelot) entity).setCatType(Ocelot.Type.valueOf(conf.getString("catType")));
+        case CAT:
+            Cat cat = (Cat) entity;
+            cat.setCatType(Type.valueOf(conf.getString("catType")));
+            cat.setCollarColor(DyeColor.valueOf(conf.getString("collarColor")));
             break;
         case WOLF:
-            ((Wolf) entity).setSitting(conf.getBoolean("sitting"));
-            ((Wolf) entity).setCollarColor(DyeColor.valueOf(conf.getString("collar")));
+            Wolf wolf = (Wolf) entity;
+            wolf.setSitting(conf.getBoolean("sitting"));
+            wolf.setCollarColor(DyeColor.valueOf(conf.getString("collarColor")));
             break;
         case HORSE:
-            Horse h = (Horse) entity;
-            h.setColor(Horse.Color.valueOf(conf.getString("horseColor")));
-            h.setVariant(Horse.Variant.valueOf(conf.getString("horseVariant")));
-            h.setStyle(Horse.Style.valueOf(conf.getString("horseStyle")));
-            h.setCarryingChest(conf.getBoolean("chest"));
-            h.setJumpStrength(conf.getDouble("jumpStrength"));
-            h.setDomestication(conf.getInt("domestication"));
-            h.setMaxDomestication(conf.getInt("maxDomestication"));
-            // separate saddle & armor entries are obsolete now
-            if (conf.contains("saddle")) {
-                h.getInventory().setSaddle(new ItemStack(Material.getMaterial(conf.getString("saddle"))));
-            }
-            if (conf.contains("armor")) {
-                h.getInventory().setArmor(new ItemStack(Material.getMaterial(conf.getString("armor"))));
+        case MULE:
+        case DONKEY:
+        case ZOMBIE_HORSE:
+        case SKELETON_HORSE:
+            if (entity instanceof Horse) {
+                Horse h = (Horse) entity;
+                h.setColor(Horse.Color.valueOf(conf.getString("horseColor")));
+                h.setStyle(Horse.Style.valueOf(conf.getString("horseStyle")));
+            } else if (entity instanceof ChestedHorse) {
```

Comment:
The `else if` should go on the next line after the bracket to be consistent with the rest of the file.